cfgov/jinja2/v1/_includes/molecules/content-image.html

```diff
@@ -0,0 +1,87 @@
+{# ==========================================================================
+
+    Image Inset
+
+    ==========================================================================
+
+    Description:
+
+    Creates image inset markup when given:
+
+    value:                Object defined from a StreamField block.
+
+    value.image_width:    String representing the image width.
+                          Possible values: (170, 270, 470).
+                          Defaults to 270.
+
+    value.image_position: String indicating how to float the image.
+                          Possible values: (left, right).
+                          Defaults to right.
+
+    value.text:           String of caption text to be displayed below image.
+
+    value.image:          An ImageBasic atom object containing:
+
+    value.image.upload:   A CFGOVImage object.
+
+    value.is_bottom_rule: Boolean indicating whether the inset has a
```

Comment:
should this be prefixed with `has_` not `is_`?

Reply:
It _should_, but that's not how it's defined in the Wagtail model, and renaming the field would necessitate a data migration.